packages/ia-topnav/src/styles/ia-topnav.js

```diff
@@ -0,0 +1,42 @@
+import { css } from 'lit-element';
+
+export default css`
+  :host {
+    --white: #fff;
+    --grey13: #222;
+    --grey20: #333;
+    --grey28: #474747;
+    --grey999: #999;
+    --grey80: #ccc;
+    --black: #000;
+    --link-color: #428bca;
+    --subnavLinkColor: #aaa;
+    --primary-text-color: var(--white);
+    --activeColor: var(--white);
```

Comment:
I know the use of CSS variables for colors is a hot topic. It was for my past teams as well until we realized that there are really *two* needs for variables:

1. We never want raw hex numbers for colors, cuz they're meaningless and error prone, so we need good descriptive names, e.g. "scarlet" and "grey999". These variable values should never change.
2. We want variables to assign colors to meaningful contexts, e.g. "defaultLinkColor" or "topNavBackground". These values change as design demands change.

The key decoupling was that variables of type 2 should be set to variables of type 1. So setting `--white: #fff` and then `--activeColor: var(--white)` is a perfect example of this.

I've found it helpful to group the two kinds of variables together, so it's clear which ones are "color constants" and which are "design assignments".